README.md

```diff
@@ -46,4 +46,6 @@ requires = ["setuptools", "setuptools-protobuf"]
 [tool.setuptools-protobuf]
 mypy = true
 protobufs = ["example/foo.proto"]
+# srcdir is used as the cwd for the protoc commands. Defaults to the project root if not specified.
+srcdir = "src"
```

Comment:
Rather than having the user specify this twice, shouldn't we be able to retrieve this directory from setuptools internals?

Reply:
I agree and in theory we should be able to use the `package_dir` variable in the Distribution object. This should be trivial to implement in the src and flat layouts, but if package uses a complex layout with multiple entries in `package_dir` it's not obvious which one should be used for the protobuf generation. For example, someone might have:
```
package_dir = { "package1" : "../some_dir", 
                          "package2": "../some_other_dir",
                          "main_package" : "src" }
```

One solution would be to change the `protobufs` config to a dict so we can associate each set of protobufs with a package. This would be a breaking change for anyone using this package now though. For example:
```
[tool.setuptools-protobuf]
protobufs = { "package1" : [ "example/foo.proto" ],
                     "package2" : [ "proto/bar.proto" ] }
```

The other option is to iterate through each of the entries in the `package_dir` dict and search for the files that exist in each directory. This could actually be quite nice, but might lead to some ambiguities if a .proto file exists in multiple packages with the same name.

I'm going to start working on the second option, but let me know if you have any suggestions or preferences. Oh, and pardon my ignorance if I'm missing anything - this is my first foray into Setuptools plugins.

Reply:
Supporting just the flat and src layouts (which are by far the most common I believe) without additional config config seems like a reasonable approach for now.

I like this suggestion as an alternative, though ideally only people with odd layouts would have to specify these options, with the rest just providing a simple list.

```
[tool.setuptools-protobuf]
protobufs = { "package1" : [ "example/foo.proto" ],
                     "package2" : [ "proto/bar.proto" ] }
```